server/container/guice/cassandra-ldap-guice/src/test/java/org/apache/james/CassandraLdapJamesServerTest.java

```diff
@@ -19,6 +19,9 @@
 
 package org.apache.james;
 
+import static org.apache.james.user.ldap.DockerLdapSingleton.JAMES_USER;
+import static org.apache.james.user.ldap.DockerLdapSingleton.LIMIT_TO_10_MESSAGES;
```

Comment:
`LIMIT_TO_10_MESSAGES` is a JMAP related constant, it has nothing to do in `DockerLdapSingleton`